src/client/utils/html.js

```diff
@@ -155,8 +171,9 @@ export function cleanUpHtml (html) {
             }
         });
 
-        find(container, '[' + INTERNAL_ATTRS.hoverPseudoClass + ']', el => {
+        find(container, '[' + INTERNAL_ATTRS.hoverPseudoClass + '],[' + INTERNAL_ATTRS.focusPseudoClass + ']', el => {
```

Comment:
Move all calculated selectors to contants.
```js
'[class*="' + SHADOW_UI_CLASSNAME.postfix + '"]',
'[' + INTERNAL_ATTRS.hoverPseudoClass + '],[' + INTERNAL_ATTRS.focusPseudoClass + ']',
`${FAKE_TAG_NAME_PREFIX}head, ${FAKE_TAG_NAME_PREFIX}body`
```

It is perfomance optimization.
Before selectors is calculated on each `cleanHtml` call.
After it will be calculated only on module creation.